chrome/skin/style.css

```diff
@@ -151,3 +151,12 @@ popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
 #universal-search-recommendation.highlight #universal-search-recommendation-url {
   color: HighlightText;
 }
+#universal-search-recommendation.highlight #universal-search-recommendation-image {
+  background-color: #FFF;
+  box-shadow: 0;
```

Comment:
Oh, this is conditionally applied when the highlight is present, I see. Let me play with the branch to look for jank due to z-index recalculation, but this is probably fine.